spec.emu

```diff
@@ -13,22 +13,73 @@ contributors:
   - Leo Balter
   - Caridy Patiño
   - Rick Waldron
+  - Ashley Claymore
 </pre>
 
-<emu-clause id="sec-isobjectorsymbol" aoid="IsObjectOrSymbol">
-  <h1>IsObjectOrSymbol ( _argument_ )</h1>
-  <p>The abstract operation IsObjectOrSymbol takes argument _argument_ (an ECMAScript language value) and determines if its Type is Object or Symbol. This operation performs the following steps when called:</p>
+<emu-clause id="sec-ecmascript-data-types-and-values" aoid="Type">
+  <h1>ECMAScript Data Types and Values</h1>
+  <emu-clause id="sec-object-type">
+    <h1>The Object Type</h1>
+    <emu-clause id="sec-well-known-intrinsic-objects">
+      <h1>Well-Known Intrinsic Objects</h1>
+      <emu-table id="table-well-known-intrinsic-objects" caption="Well-Known Intrinsic Objects" oldids="table-7">
+        <table>
+          <tr>
+            <th>
+              Intrinsic Name
+            </th>
+            <th>
+              Global Name
+            </th>
+            <th>
+              ECMAScript Language Association
+            </th>
+          </tr>
+          <tr>
+            <td>
+              <ins>%CanBeHeldWeakly%</ins>
+            </td>
+            <td>
+            </td>
+            <td>
+              <ins>A function to check if a value can be held weakly (<emu-xref href="#sec-%canbeheldweakly%"></emu-xref>)</ins>
+            </td>
+          </tr>
+        </table>
+      </emu-table>
+    </emu-clause>
+  </emu-clause>
+</emu-clause>
+<emu-clause id="sec-%canbeheldweakly%">
+  <h1>%CanBeHeldWeakly%( _argument_ )</h1>
```

Comment:
We need to explicitly define `.name`, otherwise it will probably be missing or `"%CanBeHeldWeakly%"` (I never read the part of the spec that automatically defines function names). Or you could define it as an "exposed function" directly, similarly to how `Array.prototype.values` is defined normally and then `Array.prototype.@@iterator` "copies" it.

Reply:
I should have read 10 lines below before commenting :upside_down_face: